src/layouts/Layout.astro

```diff
@@ -2,24 +2,22 @@
 interface Props {
   title: string;
   description: string;
-
 }
 
 const { title, description } = Astro.props;
-
 ---
 
 <!doctype html>
-<html lang="en">
+<html lang="en" class="scroll-smooth">
   <head>
     <meta charset="UTF-8" />
-    <meta name="description" content= {description}/>
+    <meta name="description" content={description} />
     <meta name="viewport" content="width=device-width" />
     <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
     <meta name="generator" content={Astro.generator} />
     <title>{title}</title>
   </head>
-  <body>
+  <body class="md:overflow-auto">
```

Comment:
is this needed?

Reply:
I add it because when I open navigation on mobile and then switch resolution higher than md, scrolling is blocked. Maybe i need to find a better solution of this...

Reply:
```css
body[navigation-open=true] {
  @apply ...
}

Reply:
Applied